specification/src/main/asciidoc/chapters/Sparkplug_4_Topics.adoc

```diff
@@ -201,44 +255,109 @@ Immediately upon reception of an Edge Node Death Certificate, any MQTT client su
 Edge Node should set the data quality of all metrics to STALE and should note the time stamp when
 the NDEATH message was received.
 
+[[topics_death_message_ndeath]]
+====== Topic (NDEATH)
+
+The Death Certificate topic for an Sparkplug Edge Node is:
+[subs="quotes"]
+  namespace/group_id/*NDEATH*/edge_node_id
+  
+[[payloads_desc_ndeath]]
+====== Payload (NDEATH)
+
+The NDEATH message contains a very simple payload that only includes a single metric, the bdSeq number, so 
```

Comment:
Maybe turn this into a normative statement including 'must'?

Reply:
Done